.github/workflows/github-ci.yml

```
@@ -72,7 +72,58 @@ jobs:
         with:
           extra_args: --all-files
 
-  default-build:
+  windows-release-build:
+    name: Windows release
```

```suggestion
    name: Windows release
    needs: [format]
```
Is this job supposed to require the code be formatted beforehand like the linux builds?